CHANGELOG.rst

```diff
@@ -4,6 +4,231 @@
+
+    Deprecate the ``techniques`` property in favor of the ``uses`` light edge.
+
+  ``risk:tool:software``
+    Add a ``availability`` property to record the assessed availability of the
+    tool
+
+    Add a ``sophistication`` property to record the assessed sophistication of
+    the software.
+
+    Migrate the ``type`` property to ``risk:tool:software:taxonomy``.
+
+    Deprecate the ``techniques`` property in favor of the ``uses`` light edge.
+
+  ``risk:tool:software:taxonomy``
+    Rename the type ``risk:tool:taxonomy`` to ``risk:tool:software:taxonomy``.
+
+  ``risk:vuln``
+    Add a ``mitigated`` property to record if a mitigation or fix is avilable
+    for the vulnerability.
+
+    Add a ``exploited`` property to record if the vulnerability has been
+    exploited in the wild.
+
+    Add ``timeline:discovered``, ``timeline:published``,
+    ``timeline:vendor:notified``, ``timeline:vendor:fixed``, and
+    ``timeline:exploited`` properties to record the timeline for significant
+    events on a vulnerability.
+
+    Add ``cve:desc``, ``cve:url``, and ``cve:references`` secondary properties
+    to record information about the CVE associated with a vulnerability.
+
+    Add ```nist:nvd:source`` to record the name of the organization which
+    reported the vulnerability in the NVD.
+
+    Add ``nist:nvd:published`` and ``nist:nvd:modified`` to record when the
+    vulnerability was first published, and later modified, in the NVD.
+
+    Add ``cisa:kev:name``, ``cisa:kev:desc``, ``cisa:kev:action``,
+    ``cisa:kev:vendor``, ``cisa:kev:product``, ``cisa:kev:added``,
+    ``cisa:kev:duedate`` properties to record information about the CISA KEV
+    database for the vulnerability.
+
+- Annotate the following light edges.
+  (`#2900 <https://github.com/vertexproject/synapse/pull/2900>`_)
+
+  ``seen``
+    When used with ``meta:source`` nodes, the edge indicates the target
+    node was observed by the source node.
+
+  ``uses``
+    When used with ``ou:campaign`` and ``ou:technique`` nodes, the edge
+    indicates the campaign used a given technique.
+
+    When used with ``ou:org`` and ``ou:technique`` nodes, the edge
+    indicates the organization used a given technique.
+
+    When used with ``risk:attack`` and ``ou:technique`` nodes, the edge
+    indicates the attack used a given technique.
+
+    When used with ``risk:threat`` and ``ou:technique`` nodes, the edge
+    indicates the threat cluster uses the technique.
+
+    When used with ``risk:tool:software`` and ``ou:technique`` nodes, the edge
+    indicates the tool uses the technique.
+
+    When used with ``risk:attack`` and ``risk:vuln`` nodes, the edge
+    indicates the attack used the vulnerability.
+
+    When used with ``risk:threat`` and ``risk:vuln`` nodes, the edge
+    indicates the threat cluster uses the vulnerability.
+
+    When used with ``risk:tool:software`` and ``risk:vuln`` nodes, the edge
+    indicates the tool used the vulnerability.
+
```

Comment:
i think you're missing a heading for 'targets' here

Reply:
That was a bit confusing; let me update that.